lib/pymedphys/_icom/patients.py

```diff
@@ -126,6 +126,10 @@ def update_data(self, ip, data):
 
                 return
 
+            # Every iCOM data item has an index, this check here is
+            # making sure that the currently received data item is
```

Comment:
Is monotonic increasing sufficient or is exact increment by one necessary?

Reply:
I'm not sure... I don't have access to a Linac anymore... But, this listener, with the monotonic increasing check, has been humming along without issue for ~2 years.